src/wp-includes/post.php

```diff
@@ -7801,6 +7801,8 @@ function _prime_post_caches( $ids, $update_term_cache = true, $update_meta_cache
  *
  * @global wpdb $wpdb WordPress database abstraction object.
  *
+ *  @since 6.4.0
```

Comment:
```suggestion
 * @since 6.4.0
```

Remove double space.